### ISSUE TITLE
Draft pages appear in search results

### ISSUE THREAD

**Issue by wmakley** [refinery/refinerycms-search#50]
Draft pages should not be shown to the general public, but they still appear in the search results. I couldn't find a good way to patch in a quick fix, so I am recording the issue here. It seems like SearchEngine or Page could provide a way to specify an initial scope for the search.

**Comment by bricesanchez** [comment]
i can reproduce your bug. Do you have find a fix ?

**Comment by wmakley** [comment]
No, in the end I think we just deleted the page because I couldn't get any hacks to work quickly (I tried to alias_method_chain "with_query", to no avail). I think a perfectly good solution would be to modify the SearchEngine class to call a method on the model (much like friendly_search_name) that the user can customize. Maybe something like:

```ruby
Refinery::Page.class_eval do
  def self.refinery_search_scope(query)
    live.with_query(query)
  end
end
```
I don't like monkey-patching Refinery::Page very much though. If I have time to come back to this and get it working, I will absolutely submit a pull request.
